src/components/containers/ArticleLists/ArticleLists.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/jsx-key */
+/* eslint-disable no-console */
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { Item, Icon } from 'semantic-ui-react';
+
+import Title from '../../presentationals/Title/Title.jsx';
+
+const f16 = {
+  fontSize: '16px',
+};
+
+class ArticleLists extends Component {
```

Comment:
We can change this to a stateless component.

Reply:
Ok, removed.